final_task/.gitignore

```diff
@@ -0,0 +1,2 @@
+*.рус
```

Comment:
You dont need to specify .gitignore in each folder. You can define it on project root. Please, add .idea to .gitignore

Reply:
Fixed.

Reply:
.idea folder is not added still